src/annotation.js

```diff
@@ -598,6 +605,8 @@ var pointAnnotation = function (args) {
 };
 inherit(pointAnnotation, annotation);
 
+registerAnnotation('point', pointAnnotation, {point: true});
```

Comment:
I'm not sure having another list aids much here.  There is no strict reason why an annotation should need to use the same feature that any other annotation used (I don't know what a heatmap annotation would look like, but we could have one).

I think if we wanted to make this more based on definitions, then we should probably add a `general` key to each base feature's capabilities map which contains the plain feature's name.  Then we would import the base feature (not the renderer-specific feature) so that we could specify one of its capabilities.